src/annotator/config/index.js

```diff
@@ -51,3 +96,23 @@ export default function configFrom(window_) {
     ),
   };
 }
+
+/**
+ * Return the configuration for a given application context.
+ *
+ * @param {'sidebar'|'notebook'|'annotator'|'all'} [appContext] - The name of the app.
+ */
+export default function getConfig(appContext = 'annotator', window_ = window) {
```

Comment:
Can you use a named rather than default export here per our recently adopted conventions.